README.md

```diff
@@ -1 +1,51 @@
-# Portfolio-Website
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+

```

Comment:
- [ ] It would be better if you would add the Title to your readme file. :+1: such as **Portfolio**. You can check out this [readme](https://github.com/HAMMAS-SALEEM/Hello-Microverse/blob/master/README.md) to get a good understanding. :blush: